Signal/src/network/PushManager.m

```diff
@@ -162,6 +162,10 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
     }
 }
 
+- (void)applicationDidBecomeActive {
```

Comment:
I also intend to start using a precommit script to do things like normalize our file headers, etc.